liblangutil/Scanner.cpp

```diff
@@ -76,7 +76,8 @@ string to_string(ScannerError _errorCode)
 		case ScannerError::IllegalHexDigit: return "Hexadecimal digit missing or invalid.";
 		case ScannerError::IllegalCommentTerminator: return "Expected multi-line comment-terminator.";
 		case ScannerError::IllegalEscapeSequence: return "Invalid escape sequence.";
-		case ScannerError::IllegalCharacterInString: return "Invalid character in string.";
+		case ScannerError::IllegalCharacterInStringSolMode: return "Invalid character in string. If you are trying to use Unicode characters, use a unicode\"...\" string literal.";
+		case ScannerError::IllegalCharacterInStringYulMode: return "Invalid character in string.";
```

Comment:
```suggestion
		case ScannerError::UnicodeCharacterInNonUnicodeString: return "Invalid character in string. If you are trying to use Unicode characters, use a unicode\"...\" string literal.";
		case ScannerError::IllegalCharacterInString: return "Invalid character in string.";
```
Maybe better to choose names according to the intention of the messages rather than the scanner mode?